src/components/HelloWorld.vue

```diff
@@ -0,0 +1,48 @@
+<script lang="ts">
```

Comment:
This file can be removed.

Reply:
There is a exemple Unit Test using this component. Probably in the next update (intro to testing) this will be removed